README.md

```diff
@@ -541,6 +542,29 @@ end
 
 [grape-swagger]: https://github.com/ruby-grape/grape-swagger
 
+## Configuration
+
+Grape counts with a module `Grape.configure` for some basic configuration at load time.
```

Comment:
Counts is probably the wrong word. I tend to keep this as short as possible. For example:

_Use `Grape.configure` to setup global settings at load time._